### ISSUE TITLE
Training loss explode when training on my own dataset

### ISSUE THREAD

**Issue by msxiaojin** [SHI-Labs/Cross-Scale-Non-Local-Attention#15]
Hi, thanks for your work!
I tried to train the model using my own dataset. I used the default setting except the batchsize. I used batchsize=4 to avoid GPU memory error. 

However during training, the training loss suddenly went to Inf after 2 epochs. Have you encounted similar problem? If so, how did you solve it?

Best regards.

**Comment by HarukiYqM** [comment]
Hi, please try to reduce patch size and keep batch size at least 8.

**Comment by lidong-pig** [comment]
[](https://user-images.githubusercontent.com/93692234/160311266-ffa9683b-edc5-41cc-bd8a-98fb16311ed5.png)你好！应该将 div2k 数据集在哪里？我每次运行它会报错创建一个 div2kbin/div2k_train-hr 和 LR 文件夹。不是我运行时应该将 800张样本 训练 bin/train-hr
图片
